### PR TITLE
DOCS-847: add mention of orin nano to board model docs for jetson

### PR DESCRIPTION
**Pull request by sguequierre**
* came up from https://viam.atlassian.net/browse/DOCS-843 that although Jetson Nano support is uncertain, Orin Nano should be on there
* adds across boards docs, not to carousel (already 3 nvidia boards on there)

### CODE REVIEW

File: docs/components/board/_index.md [hunk 1]
```diff
@@ -35,8 +35,8 @@ For model-specific configuration information, click on one of the following mode
...
 | [`ti`](ti/) | [Texas Instruments TDA4VM](https://devices.amazonaws.com/detail/a3G8a00000E2QErEAN/TI-TDA4VM-Starter-Kit-for-Edge-AI-vision-systems) |
-| [`beaglebone`](beaglebone/) | [BeagleBoard's BeagleBone AI 64](https://beagleboard.org/ai-64) |
-| [`jetson`](jetson/) | [NVIDIA Jetson AGX Orin](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Xavier NX](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/), [NVIDIA Jetson  Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-nano/) |
+| [`beaglebone`](beaglebone/) | [BeagleBoard's BeagleBone AI-64](https://beagleboard.org/ai-64) |
+| [`jetson`](jetson/) | [NVIDIA Jetson AGX Orin](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Orin Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Xavier NX](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/), [NVIDIA Jetson Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-nano/) |
```
Comment (penguinland) [inline docs/components/board/_index.md:39]: :exploding_head: I wrote a comment saying to change "AGX Orin" to "Orin AGX," then went to double-check that. You've got it right here, and I've been saying it wrong for 9 months! I wonder if I've influenced any of the documentation that is now wrong...

Comment (penguinland) [reply docs/components/board/_index.md:39]: I got surprised that the AGX Orin and Orin Nano have the same URL, but having visited that website, yes, the URL is for both of them!

Comment (penguinland) [reply docs/components/board/_index.md:39]: The Jetson Xavier NX links to the Jetson AGX Xavier page. I wonder if https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-agx-xavier/ is supposed to be https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-xavier-nx/

**Review by penguinland — APPROVED** [review]
Not sure about the Jetson Xavier link, but the rest LGTM!

File: docs/components/board/_index.md (hunk 1 above)
Comment (sguequierre) [inline docs/components/board/_index.md:39]:
```suggestion
| [`jetson`](jetson/) | [NVIDIA Jetson AGX Orin](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Orin Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-orin/), [NVIDIA Jetson Xavier NX](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-xavier-nx/), [NVIDIA Jetson Nano](https://www.nvidia.com/en-us/autonomous-machines/embedded-systems/jetson-nano/) |
```

**Pull request merged**